### PR TITLE
Add departments to admin menu

### PR DESCRIPTION
**Pull request by sampatbadhe**
Hello @kgf

This PR will address - https://github.com/ReadyResponder/ReadyResponder/issues/230

Please review and advice.

Thanks

### CODE REVIEW

**Comment by kgf** [comment]
@sampatbadhe thanks for your help. Sadly, this one failed Travis

**Comment by sampatbadhe** [comment]
@kgf I have fixed the failing specs, also added more specs. please review and let me know if any changes required.

File: db/seeds.rb [hunk 1]
```diff
@@ -27,6 +27,7 @@
 Role.find_or_create_by(name: "Depositor")
 Role.find_or_create_by(name: "Trainer")
 Role.find_or_create_by(name: "Admin")
+Role.find_or_create_by(name: "Reader")
```
Comment (kgf) [inline db/seeds.rb:30]: Nice ! Always appreciate updates to the seeds.rb

File: spec/features/departments_spec.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.feature "Department" do
+  describe "user in the editor role" do
+    before do
+      somebody = create(:user)
```
Comment (kgf) [inline spec/features/departments_spec.rb:6]: I'm going to merge this, so please don't change it now, but if you are willing to help more, @david135 has provided a helper for this logging in boilerplate.

File: spec/features/departments_spec.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      @department = create(:department)
+      sign_in_as('Reader')
+    end
+
+    it "cannot edit department" do
+      visit edit_department_path(@department)
+      expect(page).to have_content("Access Denied")
+    end
+
+    it "cannot create a new department" do
```
Comment (kgf) [inline spec/features/departments_spec.rb:50]: I like the tests for the "Unhappy" path

**Comment by kgf** [comment]
Thanks for your help !

**Pull request merged**